Sources/Defaults/PublishedDefault.swift

```diff
@@ -0,0 +1,86 @@
+import Combine
+
+@propertyWrapper
+public struct PublishedDefault<Value: _DefaultsSerializable> {
+    private let key: Defaults.Key<Value>
+
+    private var value: Value {
+        get { Defaults[key] }
+        set { Defaults[key] = newValue }
+    }
+
+    /**
+     Get/set a `Defaults` item and also trigger `objectWillChange` in the `ObservableObject` when the value changes.
+
+     - Important: Like `@Published`, `@PublishedDefault` does not observe the change of the corresponding value. Only changes made via this `@PublishedDefault` will trigger `ObservableObject`'s `objectWillChange`.
+
+     ```swift
+     extension Defaults.Keys {
+         static let opacity = Key<Double>("opacity", default: 1)
+     }
+
+     class ViewModel: ObservableObject {
+         @PublishedDefault(.opacity) var opacity
+     }
+     ```
+     */
```

Comment:
```suggestion
    */
```

Incorrect indentation on the whole comment block.

Reply:
BTW, how (or what tool did you use) to adjust only the comment block's indentation width?

Reply:
This:

<img width="321" alt="Screenshot 2023-03-09 at 14 30 26" src="https://user-images.githubusercontent.com/170270/223951302-25d948c8-b8ae-4ff6-9a14-4935a8a24fa5.png">

Reply:
I mean, on Xcode.

On Xcode `/** */` block, when you hit enter key, the new line will start at 5 space position.
Did you adjust it manually every time?

Reply:
Yup...